rama-http/src/headers/x_robots_tag_components/custom_rule.rs

```diff
@@ -0,0 +1,28 @@
+use crate::headers::util::value_string::HeaderValueString;
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub(super) struct CustomRule {
+    key: HeaderValueString,
+    value: Option<HeaderValueString>,
+}
+
+impl CustomRule {
+    pub(super) fn as_tuple(&self) -> (&HeaderValueString, &Option<HeaderValueString>) {
```

Comment:
```suggestion
    pub(super) fn as_tuple(&self) -> (&HeaderValueString, Option<&HeaderValueString>) {
```

Usually you move the reference inwards. For this to happen you'll need to change in the line below `&self.value` to `self.value.as_ref()`